README.md

```diff
@@ -22,3 +22,5 @@ Examples of using EvolviConf can be found in the [examples](/examples)
 directory.
 
 EvolviConf was created and open-sourced by [Meroxa](https://meroxa.io).
+
+![scarf pixel](https://static.scarf.sh/a.png?x-pxid=c07050b2-6ffc-4793-b05f-5d7b9d0cf34e)
```

Comment:
@raulb  for future references, we should refer to the actual pixel name between the brackets (`evolviconf-readme` instead of `scarf pixel`), so it's easier for us to check it or update it when needed